plugins/webapi/message/plugin.go

```diff
@@ -125,9 +125,10 @@ func GetMessageMetadata(c echo.Context) (err error) {
 
 // NewMessageMetadata returns MessageMetadata from the given tangle.MessageMetadata.
 func NewMessageMetadata(metadata *tangle.MessageMetadata) jsonmodels.MessageMetadata {
-	branchID, err := deps.Tangle.Booker.MessageBranchID(metadata.ID())
-	if err != nil {
-		branchID = ledgerstate.BranchID{}
+	var branchID ledgerstate.BranchID
+	branchIDs, err := deps.Tangle.Booker.MessageBranchIDs(metadata.ID())
+	if err == nil {
+		branchID = ledgerstate.NewAggregatedBranch(branchIDs).ID()
```

Comment:
This sequence seems to be used more frequently. Maybe we can have a function that does this kind of thing for us? Possibly together with the branch alias / compression refactor